### ISSUE TITLE
Review normalization

### ISSUE THREAD

**Issue by miguelriemoliveira** [lardemua/atom#271]

**Comment by eupedrosa** [comment]
This was reviewed and fixed by me and @miguelriemoliveira.

**Comment by aaguiar96** [comment]
Hi @miguelriemoliveira  and @eupedrosa 

I am curious. Can you summarize what was the issue?

**Comment by eupedrosa** [comment]
TLDR; The calculation of the  `noisy initial guess` was wrong. The optimization failed due to super duper uber unrealistic initial guesses.
